flutter/lib/ui/settings_screen.dart

```diff
@@ -146,6 +146,17 @@ class _SettingsScreen extends State<SettingsScreen> {
               }
             },
           ),
+          ListTile(
+            title: Text(
+              stringResources.clearCache,
+              style: TextStyle(color: Colors.blue),
```

Comment:
I think it's confusing that element that looks like just blue text and is placed near other text elements in settings acts like a button.
Can we use a real button?

Reply:
I oriented to iOS Settings app, which uses blue text for reset action.
But I changed to use a `TextButton` now. I also added a confirm dialog.